### PR TITLE
feat(troubleshooting): add Internal Server Errors section

### PR DESCRIPTION
**Pull request by joshtrichards**
* New section: "Internal server errors"
* Refactor section: "Nextcloud log files"

### 🖼️ Screenshots

image

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
LGTM, thanks :)

File: admin_manual/issues/general_troubleshooting.rst [hunk 1]
```diff
@@ -51,14 +51,37 @@ different issues. Always disable 3rd party apps before upgrades, and for
...
+
+.. tip:: When asking for help, the entire raw log entry is generally required.
+
+.. note:  In a standard Nextcloud installation the log level is set to ``2``. This is 
+    known as the ``WARN`` level. It is sufficient for catching for day-to-day problems 
+    (warnings, errors, and fatal errors).
+
+For some situations you may need to adjust the log level in your ``config.php``
+file. Please see :doc:`../configuration_server/logging_configuration` for more 
+information on these log levels.
```
Comment (szaimen) [inline admin_manual/issues/general_troubleshooting.rst:84]: btw, nowadays one can adjust the loglevel via the logreader app...

**Review by tflidd — APPROVED** [review]

**Pull request merged**